doc/src/reference.xml

```diff
@@ -561,13 +561,15 @@ path-constant DATA : data/a.txt ;
       <varlistentry>
         <term><literal>cflags</literal></term>
         <term><literal>cxxflags</literal></term>
+        <term><literal>compileflags</literal></term>
```

Comment:
I do not see how writing that 'cflags' applies to both C and C++ can be correct. Clearly this is not the case for any compiler toolset that is documented. Leaving that in the documentation is just a cause for confusion. It had certainly confused me until I realized it wasn't tthe case for any compiler toolset which is documented in Boost Build.